assemblies/assembly_managing-business-central/main.adoc

```diff
@@ -0,0 +1,91 @@
+[id='assembly_managing-business-central']
+
+include::_artifacts/document-attributes.adoc[]
+
+= Managing {CENTRAL}
+// Context attribute is assembly specific and enables module reuse between assemblies.
+:doctype: book
+:imagesdir: _images
+:context: managing-business-central
+include::_artifacts/author-group.adoc[]
+
+// Purpose statement for the assembly
+As an administrator, you can customize {CENTRAL} using the `Admin Settings`. You can use the *Admin Settings* to customize the following items:
+
+* `Roles`: View and set the home page, priority, and permissions of a role.
```

Comment:
If by `View` you refer to the left-side panel where the roles are being listed and can be searched, I would recommend excluding this here, because this is in a way implied. You can't do any setting change for a role if you can't search and view them beforehand.
The other thing is that listing the `priority` just as another setting among others, doesn't really put this setting in the proper context. It is not just another independent setting for the role, but is rather a parameter which sets the place of all other settings made for the role in the overall system-wide hierarchy of roles.
Hence, I would rephrase this to something along the lines of:
`Set the home page and permissions for a role, and set the priority of the settings made.`
The same comment goes for `Groups` and `Users`. In the `Groups`, you even have `view` mentioned twice which mean the same thing.

Reply:
changed.